src/api/useElmTranslationServiceApi.ts

```diff
@@ -0,0 +1,79 @@
+import axios from "axios";
+import useServiceConfig from "./useServiceConfig";
+import { ServiceConfig } from "./ServiceContext";
+import useOktaTokens from "../hooks/useOktaTokens";
+
+export type ElmTranslationError = {
+  startLine: number;
+  startChar: number;
+  endChar: number;
+  endLine: number;
+  errorSeverity: string;
+  errorType: string;
+  message: string;
+  targetIncludeLibraryId: string;
+  targetIncludeLibraryVersionId: string;
+  type: string;
+};
+
+export type ElmTranslationLibrary = {
+  annotation: any[];
+  contexts: any;
+  identifier: any;
+  parameters: any;
+  schemaIdentifier: any;
+  statements: any;
+  usings: any;
+  valueSets: any;
+};
+
+export type ElmTranslation = {
+  errorExceptions: ElmTranslationError[];
+  externalErrors: any[];
+  library: ElmTranslationLibrary;
+};
+
+export class ElmTranslationServiceApi {
+  constructor(private baseUrl: string, private getAccessToken: () => string) {}
+
+  async translateCqlToElm(cql: string): Promise<ElmTranslation> {
+    if (this.baseUrl) {
+      const resp = await axios.put(`${this.baseUrl}/cql/translator/cql`, cql, {
```

Comment:
this is an interesting URL

Reply:
This code is something related to next ticket, I forgot to push the code before working on the next ticket. I'll remove this file it is not related to this ticket